roles/backup/tasks/main.yml

```diff
@@ -1,47 +0,0 @@
---- # Backup
-
-#--------------------
-# Get timestamp
-
-- name: get current date for timestamp
-  shell: date +"%Y-%m-%d-%I%M%S"
-  register: current_date
-  tags:
-    - backup
-    - skip_ansible_lint
-
-#-----------------
-# Backup and fetch
-# @TODO cleanup old backups
-
-# Create a new database backup every time
-- name: backup remote database
-  shell: pg_dump -h {{ db_host }} -U {{ db_user }} {{ db }} | gzip > {{ backup_path }}/{{ backup_filename }}
-  become: yes
-  become_user: "{{ unicorn_user }}"
-  tags: skip_ansible_lint
-
-- name: fetch database to local
-  fetch:
-    src={{ backup_path }}/{{ backup_filename }}
-    dest={{ local_backup_path }}/{{ domain }}/databases/
-    fail_on_missing=yes
-    flat=yes
-
-# But we don't bother making a whole new copy of all the files every time, just rsync them.
-- name: sync public files to local
-  synchronize:
-    mode=pull
-    src={{ current_path }}/public
-    dest={{ local_rsync_path }}/{{ domain }}/files
-    copy_links=yes
-  tags: rsync
```

Comment:
I thought the backup was backing up something more than the DB ☝️ 
is it still the case?

Reply:
I was quite pleased to :fire: this code, tbh. I don't think it's in use, and it's pretty messy. We can perform the same tasks with the new playbooks I've added today, we're not losing anything.

Reply:
ok :+1: